lua/telescope/_extensions/node_modules_builtin.lua

```diff
@@ -103,7 +103,9 @@ M.list = function(opts)
   )
 
   local info = package_info(opts.cwd)
-  if not info then error('cannot find package.json') end
+  if not info then
+    vim.notify("package.json not found", vim.log.levels.WARN)
```

Comment:
`vim.notify` does not throw exceptions unlike `error`, so it does not stop the process. Maybe, it will die at line 109 because `info` is `nil`, I think.

```suggestion
    vim.notify("package.json not found", vim.log.levels.WARN)
    return
```

↑ This makes good even in such case, don't you think?

Reply:
My bad, should have tested this!